test/Microsoft.AspNetCore.Razor.Language.Test/IntegrationTests/CodeGenerationIntegrationTest.cs

```diff
@@ -23,12 +24,6 @@ public void CSharp7_Runtime()
             RunTimeTest();
         }
 
-        [Fact]
-        public void BasicImports_Runtime()
```

Comment:
The way this test is authored isn't really a good fit for a baseline test. Our default configuration doesn't even **have** imports, and the old test infrastructure kind jams them in there. I don't really feel like there was anything novel here.